todo.py

```diff
@@ -0,0 +1,31 @@
+from flask_pymongo import PyMongo
+from flask import Flask , request , jsonify , abort
+
+app = Flask(__name__)
+mongodb_client = PyMongo(app, uri="mongodb://localhost:27017/todo_db")
+db = mongodb_client.db
+
+
+@app.route('/todo',methods=["POST"])
+def add_todo():
+    if not request.json:
+        abort(500)
+
+    id = request.json.get("id", None)
```

Comment:
You never send the id in the request body, It should be unique, And generated automatically.